scripts/tuple_tree_generator/tuple_tree_generator/templates/struct.h.tpl

```diff
@@ -123,6 +123,22 @@ public:
       /**- endif -**/
     }
 
+  /** if struct.abstract **/
```

Comment:
@aleclearmind I added this to the PR, please take a look (the very last commit: `Model: forbid copies and moves of abstract structs`). Shouldn't be controversial, but just in case.

Reply:
If the code still compiles, seems reasonable!